platform/spire/src/setup.py

```diff
@@ -141,6 +145,31 @@ def setup_keyserver(ops: Operations) -> None:
         ops.ssh("enable keyserver on @HOST", node, "systemctl", "enable", "keyserver.service")
         ops.ssh("start keyserver on @HOST", node, "systemctl", "restart", "keyserver.service")
 
+def redeploy_keyserver(ops: Operations) -> None:
+    config = configuration.get_config()
+    for node in config.nodes:
+        if node.kind != "supervisor":
+            continue
+        # delete the existing configs
+        ops.ssh_rm("delete existing cluster config from @HOST", node, STATICS_DIR + "/cluster.conf")
+        ops.ssh_rm("delete existing keyserver config from @HOST", node, CONFIG_DIR + "/setup.yaml")
+        # redeploy new config
+        ops.ssh_upload_bytes("reupload cluster config to @HOST", node,
+            configuration.get_cluster_conf().encode(), STATICS_DIR + "/cluster.conf")
+        ops.ssh_upload_path("upload cluster setup to @HOST", node,
```

Comment:
any reason for one message to say "reupload" vs "upload"?

Reply:
There was no reason that it should have said "upload", I have changed the offending line to "reupload". Good catch!